public/css/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+    margin: 30px;
+    padding: 20px;
+    font-family: Roboto;
+
+}
+.name{
+    margin-bottom: 4%
+}
+h3{
+    margin-bottom: 4%  
```

Comment:
so you want all your `h3` element on each page to have a `margin-bottom: 4%;` rule? is your friend ok with that?
do they want the same on their pages because it'll affect them all
@amd711   are you ok with this?